packages/MJServer/src/resolvers/EntityRecordNameResolver.ts

```diff
@@ -1,15 +1,15 @@
-import { Metadata, PrimaryKeyValue } from '@memberjunction/core';
+import { Metadata, CompositeKey } from '@memberjunction/core';
 import { Arg, Ctx, Field, InputType, ObjectType, Query, Resolver } from 'type-graphql';
 import { AppContext } from '../types';
-import { PrimaryKeyValueInputType, PrimaryKeyValueOutputType } from './MergeRecordsResolver';
+import { CompositeKeyInputType, CompositeKeyOutputType } from './PotentialDuplicateRecordResolver';
```

Comment:
@JS-BC  This is correct here, CompositeKeyInputType, Be consistent in work, above you have keyvalue arrays but here you have it right. Why there differences?